### PR TITLE
buy-shirt.cy.ts file

### PR DESCRIPTION
**Pull request by edopore**
Hi, hope you're well
I made CSS-Selectors Part

### CODE REVIEW

File: cypress/e2e/buy-shirt.cy.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    cy.get("#SubmitLogin").click();
+    // Step 9
+    cy.get("[name='processAddress']").click();
+    // Step 10
+    cy.get("#uniform-cgv").click();
+    // Step 11
+    cy.get("[name='processCarrier']").click();
+    // Step 12
+    cy.get(".bankwire").click();
+    // Step 13
```
Comment (kliver98) [inline cypress/e2e/buy-shirt.cy.ts:25]: Please delete those comments, are not useful and makes the code not pretty readable

File: cypress/e2e/buy-shirt.cy.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    cy.get("#email").type("aperdomobo@gmail.com");
+    cy.get("#passwd").type("WorkshopProtractor");
+
+    // Debes completar la prueba ...
+    // Step 8
+    cy.get("#SubmitLogin").click();
+    // Step 9
+    cy.get("[name='processAddress']").click();
+    // Step 10
+    cy.get("#uniform-cgv").click();
```
Comment (kliver98) [inline cypress/e2e/buy-shirt.cy.ts:20]: use the id of the input not the div
image

File: cypress.config.ts [hunk 3]
```diff
@@ -7,6 +7,8 @@ export default defineConfig({
     video: false,
     screenshotOnRunFailure: false,
     setupNodeEvents(on, config) {
+      config.defaultCommandTimeout = 20000
+      config.responseTimeout = 20000
       // modify config values examples
       // config.defaultCommandTimeout = 10000
```
Comment (kliver98) [inline cypress.config.ts:13]: delete these comments

File: cypress/e2e/buy-shirt.cy.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    // Step 11
+    cy.get("[name='processCarrier']").click();
+    // Step 12
+    cy.get(".bankwire").click();
+    // Step 13
+    cy.get("[type='submit'] > span > .icon-chevron-right").click();
```
Comment (kliver98) [inline cypress/e2e/buy-shirt.cy.ts:26]: improve this one, this is a button inside a cart_navigation

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**